functions/completed-checks/marking.service.js

```diff
@@ -0,0 +1,67 @@
+const moment = require('moment')
+const R = require('ramda')
+
+const sqlHelper = require('../lib/sql-helper')
+
+const markingService = {}
+
+/**
+ * Apply marking and populate asnwers table for the completedCheck
```

Comment:
Minor: typo in comments s/asnwers/answers/

Reply:
@jon-shipley thank you 'hawk-eyes' Jon!